disnake/__main__.py

```diff
@@ -105,7 +105,16 @@ def setup(bot):
     bot.add_cog({name}(bot))
 '''
 
+# everything that is a _cog_special_method goes here.
+# unlike other changes, this only checks that the functions are defined
+# it does not define any particular order.
+# this is a concious choice because the order of these should make sense for the developer
+# and the order they are defined internally may not translate to the best order here
 _cog_extras = """
```

Comment:
not sure about the specific ordering of all of these methods

Reply:
Seems good overall, the only issue I can see is that there's no clear end to the sections that start with a comment like `these checks ...`